doi/build.gradle

```diff
@@ -1,27 +1,23 @@
 plugins {
+    id 'idea'
+    id 'java-test-fixtures'
```

Comment:
Couldn't figure out what this is used for...

Reply:
In /src there is a testFixtures directory. It's a way to share common code between unit and int tests. Both the unit test and int-tests need to compare a resource document, and can use a common method in a testFixtures class. A resource document is the DOI document. Why the root of the document is called resource instead of doi? Ask DataCite. I think a resource can be other things besides a DOI.

Reply:
I see. It's a valid argument. I haven't had a need for it in the past because usually the initial test state (observations, files etc) are loaded from files and the comparison is implemented in the main code. But I would check with Pat since he has the last word on these system level changes.